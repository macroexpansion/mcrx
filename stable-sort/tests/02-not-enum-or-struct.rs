use stable_sort::stable_sorted;

#[stable_sorted]
fn main() {}
