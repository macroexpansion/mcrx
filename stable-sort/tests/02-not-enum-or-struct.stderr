error: expected struct or enum
 --> tests/02-not-enum-or-struct.rs:3:1
  |
3 | #[stable_sorted]
  | ^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `stable_sorted` (in Nightly builds, run with -Z macro-backtrace for more info)
